backend/utils/stt/vad.py

```diff
@@ -17,14 +17,27 @@
 
 
 def is_speech_present(data, vad_iterator, window_size_samples=256):
+    """
+    Quickly return true if speech is present in the given audio data.
+    
+    Args:
+    data (torch.Tensor): Audio data to check for speech.
+    vad_iterator (VADIterator): Silero VAD iterator.
+    window_size_samples (int): Size of each audio chunk to process.
+    
+    Returns:
+    bool: True if speech is detected, False otherwise.
+    """
     for i in range(0, len(data), window_size_samples):
         chunk = data[i: i + window_size_samples]
         if len(chunk) < window_size_samples:
             break
         speech_dict = vad_iterator(chunk, return_seconds=False)
         if speech_dict:
-            print(speech_dict)
+            vad_iterator.reset_states()
```

Comment:
@josancamon19 I could be wrong (I didn't have time to test it) but I believe that resetting the states also resets the trigger that flags this as having audio. So if you don't reset it, the next round will show it as having speech. Again, don't take my word for it but in the streaming examples for Silero they do suggest that you use this. Also, I have nearly identical code and it does work really well with my VAD value between 0.55-0.6. So I think this is correct.

Reply:
@josancamon19 I choose that approach because I want to look for isolated speech segments and don't want the detection state to carry over between audio received in the ws. Ideally, I want it to be adaptive like reset only after some seconds of silent, but on testing It, it's already good with this current approach. Is that ok?